include/seastar/coroutine/exception.hh

```diff
@@ -97,6 +98,7 @@ exception make_exception(T&& t) noexcept {
 template<typename T>
 [[nodiscard]]
 exception make_exception(T&& t) noexcept {
+    log_exception_trace();
     return exception(std::make_exception_ptr(std::forward<T>(t)));
```

Comment:
I don't understand how we can know that we want logging so deep in system code.

Reply:
Exactly like in https://github.com/scylladb/seastar/blob/23d28aba656b869367122fb83f0ccd3486fc5ae4/include/seastar/core/future.hh#L2086-L2088

We want to trace where the exception was generated and the assumption is that this is one place where the user created an exception object that is passed on up the stack, rather than being thrown.

For the throw case, we have https://github.com/scylladb/seastar/blob/23d28aba656b869367122fb83f0ccd3486fc5ae4/src/core/exception_hacks.cc#L138-L149

Reply:
Ah, trace level makes if less terrible.